tinygrad/tensor.py

```diff
@@ -423,9 +423,13 @@ def arange(start, stop=None, step=1, **kwargs):
     ```
     """
     if stop is None: stop, start = start, 0
-    assert all(isinstance(s, (int, float)) for s in (start, stop, step)), f"symbolic arange not supported {start=}, {stop=}, {step=}"
     dtype = kwargs.pop("dtype", dtypes.default_float if any(isinstance(x, float) for x in (start, stop, step)) else dtypes.default_int)
-    return (Tensor.full((math.ceil((stop-start)/step),), step, dtype=dtype, **kwargs)._cumsum() + (start - step)).cast(dtype)
+
+    if all(isinstance(s, (int, float)) for s in (start, stop, step)):
+      result = (Tensor.full((math.ceil((stop-start)/step),), step, dtype=dtype, **kwargs)._cumsum() + (start - step)).cast(dtype)
+    else:
+      result = Tensor(data=stop, dtype=dtype, **kwargs)
```

Comment:
?

Reply:
This was a placeholder that I accidentally submitted. Real attempt didn't work, so didn't bother to resubmit.